diff --git a/lib/scheduler/ue_scheduling/ue_srb0_scheduler.cpp b/lib/scheduler/ue_scheduling/ue_srb0_scheduler.cpp
index bac268873..c4642eb54 100644
--- a/lib/scheduler/ue_scheduling/ue_srb0_scheduler.cpp
+++ b/lib/scheduler/ue_scheduling/ue_srb0_scheduler.cpp
@@ -258,7 +258,7 @@ void ue_srb0_scheduler::fill_srb0_grant(ue&                        u,
 {
   static constexpr uint8_t srb0_dai = 0;
   // Allocate DL HARQ.
-  h_dl.new_tx(pdsch_slot, k1, expert_cfg.max_nof_harq_retxs, srb0_dai);
+  h_dl.new_tx(pdsch_slot, k1, 0, srb0_dai);
 
   // Fill DL PDCCH DCI.
   static const uint8_t msg4_rv = 0;
